unleashandroidsdk/build.gradle.kts

```diff
@@ -30,10 +34,10 @@ android {
 
     buildTypes {
         debug {
-
+            isMinifyEnabled = false
         }
         release {
-            isMinifyEnabled = false
+            isMinifyEnabled = true
```

Comment:
Do we want this? I don't remember what we talked about @chriswk sorry!

Reply:
I think we can't do that, since we're a library, minifying will say "Oh, I'm not being used by anyone, let's minimize the whole thing"

Reply:
Sorry, missed this comment before merging, I'll amend that in next PR

Reply:
Addressing this in https://github.com/Unleash/unleash-android/pull/61